contracts/src/Types.sol

```diff
@@ -107,6 +107,7 @@ struct Ticket {
     ParaID dest;
     Costs costs;
     bytes payload;
+    uint128 etherAmount;
```

Comment:
I'd just call this `value` to match V2 nomenclature.

```suggestion
    uint128 value;
```

Reply:
addressed in 0e40439370417481613eeccb5e70386649a10c73